### PR TITLE
Publish IS 7 v2 data

### PR DESCRIPTION
**Pull request by mpmadhavig**
Resolves https://github.com/wso2/product-is/issues/19977

### CODE REVIEW

File: benchmarks/7.0.0/v2/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# V2
+
+The the newly introduced features/improvements to Identity Server 7.0 base product were being used when getting these performance results. Please refer to the [release note](https://is.docs.wso2.com/en/latest/references/about-this-release/) for further information.
```
[the remark this answers is not in the document]
Comment (mpmadhavig) [reply benchmarks/7.0.0/v2/README.md]: fixed with 269c27aad35a39ce97af9fde5ec4478e0d48b412

File: benchmarks/README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # WSO2 Identity Server Performance Results.
 
+In this version of Performance results, the tests were executed against the default 7.0.0 base product without enabling or disabling any additional features.
```
Comment (darshanasbg) [inline benchmarks/README.md:3]: This is incorrect README.md to include this info.. It should be 7.0.0/README.md

**Comment by darshanasbg** [comment]
Also lets update the table in the benchmark/readme.md once the https://github.com/wso2/performance-is/pull/156 is merged.

File: benchmarks/README.md (hunk 2 above)
Comment (mpmadhavig) [reply benchmarks/README.md]: Fixed with debf7d98f992c6a0e295607c1b991e9452f3baa5

**Comment by mpmadhavig** [comment]
> Also lets update the table in the benchmark/readme.md once the #156 is merged.

https://github.com/wso2/performance-is/pull/165

**Review by darshanasbg — APPROVED** [review]

**Pull request merged**